build-system/compile/build-constants.js

```diff
@@ -18,7 +18,7 @@ const {VERSION} = require('./internal-version');
 
 const testTasks = ['e2e', 'integration', 'visual-diff', 'unit', 'check-types'];
 const isTestTask = testTasks.some((task) => argv._.includes(task));
-const isForTesting = argv.fortesting || isTestTask;
+const isForTesting = argv.fortesting || !argv._.includes('dist');
```

Comment:
I'm out of my depth here but will this set `getMode().localDev` to `true` or `getMode().testing` to true or both?

Reply:
It will allow `getMode().localDev` to be true. Without `fortesting==true`, the check early exits.
The naming of the variables is really not helping here.

Reply:
Seeing if it makes it more clear here: https://github.com/ampproject/amphtml/pull/35336

Reply:
Nice, I think the renaming helps. Thanks Jake.